tests/unit/test_security.py

```
@@ -295,15 +295,47 @@ async def test_execute_blocked_command(self):
+
         manager = PermissionManager()
         executor = SecureShellExecutor(manager)
 
-        success, stdout, stderr = await executor.execute(
-            "pwd", working_dir=str(temp_dir)
-        )
+        # Use platform-appropriate command to get current directory
+        if platform.system() == "Windows":
+            cmd = "cd"
+        else:
+            cmd = "pwd"
```

Ruff (0.11.9)

Use ternary operator `cmd = "cd" if platform.system() == "Windows" else "pwd"` instead of `if`-`else`-block

Replace `if`-`else`-block with `cmd = "cd" if platform.system() == "Windows" else "pwd"`

(SIM108)